activesupport/lib/active_support/i18n_railtie.rb

```diff
@@ -55,7 +55,7 @@ def self.initialize_i18n(app)
 
       reloader = ActiveSupport::FileUpdateChecker.new(I18n.load_path.dup){ I18n.reload! }
       app.reloaders << reloader
-      ActionDispatch::Reloader.to_prepare { reloader.execute_if_updated }
+      ActionDispatch::Reloader.to_prepare { reloader.execute_if_updated.presence }
```

Comment:
For completeness, this is how tests would fail if we let ActionDispatch callbacks halt the chain when returning `false`:

```bash
~/code/rails/railties$ ruby -w -Itest:lib:/Users/claudiob/code/rails/railties/../activesupport/lib:/Users/claudiob/code/rails/railties/../actionpack/lib:/Users/claudiob/code/rails/railties/../activemodel/lib test/application/console_test.rb
/Users/claudiob/.rbenv/versions/2.2.0/lib/ruby/gems/2.2.0/gems/bundler-1.7.9/lib/bundler/source/rubygems.rb:191: warning: shadowing outer local variable - spec
Run options: --seed 38479

# Running:

.DEPRECATION WARNING: Returning `false` in a callback will not implicitly halt a callback chain in the next release of Rails. To explicitly halt a callback chain, please use `throw :abort` instead. (called from block in test_reload_should_fire_preparation_and_cleanup_callbacks at test/application/console_test.rb:48)
F...DEPRECATION WARNING: Returning `false` in a callback will not implicitly halt a callback chain in the next release of Rails. To explicitly halt a callback chain, please use `throw :abort` instead. (called from block in test_reload_should_reload_constants at test/application/console_test.rb:72)
.

Finished in 7.202340s, 0.8331 runs/s, 4.0265 assertions/s.

  1) Failure:
ConsoleTest#test_reload_should_fire_preparation_and_cleanup_callbacks [test/application/console_test.rb:52]:
Expected: 3
  Actual: 2

6 runs, 29 assertions, 1 failures, 0 errors, 0 skips
```